src/Tribe/Image/Uploader.php

```diff
@@ -100,6 +100,9 @@ protected function upload_file( $file_url ) {
 		);
 
 		$id = wp_insert_attachment( $attachment, $upload['file'] );
+
+		require_once( ABSPATH . 'wp-admin/includes/image.php' );
```

Comment:
Do a conditional; to see `wp_generate_attachment_metadata` doesn't already exists.